data/en/mbs_0161.json

```diff
@@ -66,7 +66,7 @@
                                     ]
                                 },
                                 {
-                                    "question": "Changes in total turnover",
+                                    "question": "Changes in turnover",
```

Comment:
@nealedj, I have a feeling that this should be changes in total turnover as that's what the question it refers to is.

Reply:
Oh yeah - I see the comment above. Let's go with that.